crates/codegen/language/definition/src/model/utils/mod.rs

```diff
@@ -1,5 +1,78 @@
+use crate::model::{self, Item};
+use semver::Version;
+use std::collections::BTreeSet;
+
 mod identifier;
 mod version_specifier;
 
 pub use identifier::*;
 pub use version_specifier::*;
+
+/// Collects all versions that changes the language grammar in a breaking way.
+pub fn collect_breaking_versions(lang: &model::Language) -> BTreeSet<Version> {
+    let first = lang.versions.first().cloned().unwrap();
```

Comment:
I suggest cloning the specific version instead of the `Option<>` wrapping it:

```suggestion
    let first = lang.versions.first().unwrap().clone();
```